tools/clang/test/CodeGenDXIL/hlsl/objects/NodeObjects/mesh-node-attribs.hlsl

```diff
@@ -0,0 +1,358 @@
+// RUN: %if dxil-1-9 %{ %dxc -T lib_6_9 %s | FileCheck %s %}
```

Comment:
You can use // REQUIRES: dxil-1-9 here, no need to %if for each RUN line, since all the RUN line requirements are the same.

Reply:
I like it! Changed!